doc/sr_python_walkthrough.rst

```diff
@@ -46,17 +44,17 @@ The code below shows how to store and retrieve tensors which belong to a ``DataS
 Models
 ======
 
-The SmartRedis clients allow users to set and use a PyTorch, ONNX, TensorFlow,
+The SmartRedis client allows users to set and use a PyTorch, ONNX, TensorFlow,
```

Comment:
we actually have four clients -- one each for C, C++, Python, Fortran -- so the original statement was correct

Reply:
Ahhhhh I see!